lib/masksets.gd

```diff
@@ -0,0 +1,55 @@
+#############################################################################
+##
+#W  masksets.gd                  GAP library                     Steve Linton
+##
+##
+#Y  Copyright (C) 2017 The GAP Group
+##
+##  This file declares the operations for mask sets.
+##
+
+
+
+#############################################################################
+##
+#F  MakeMaskSet(<width>[, <width>[, <width>...]]])
+##  <#GAPDoc Label="MakeMaskSet">
+##  <ManSection>
+##  <Func Name="MakeMaskSet" Arg='width....'/>
+##
+##  <Description>
+##
+##  This function sets up the machinery for a set of bitfields of the given
```

Comment:
So why not call this a "bitfield" then? The name is well-known, at least for people who know C/C++. Whereas "MaskSet" is not (and I don't think I would have guessed what it means here).